CODEOWNERS

```diff
@@ -1 +1,2 @@
 * @nikhalim
+* @CHrislian-NI

```

Comment:
Also, I don't think they can see this repo?

![image](https://user-images.githubusercontent.com/3143819/205778495-5806d7e7-c94a-4f72-b408-28076e988cac.png)

Reply:
Hmm, I thought NI users would get access automatically. Can you help to grant the access to Chris? I don't think I have the permission to do so.

Reply:
Great! Yes, Chris has to follow the [steps listed here](https://dev.azure.com/ni/DevCentral/_wiki/wikis/AppCentral.wiki/6072/GitHub?anchor=how-do-i-get-access-to-ni-github-enterprise%3F).

Reply:
I'll change my status to approve so I don't block the PR once Chris has access (when he can see this PR, for instance).

Reply:
Chris has fixed his account issue, closing this thread.